src/rules/no-identical-functions.ts

```diff
@@ -67,11 +74,19 @@ const rule: Rule.RuleModule = {
             originalFunction.loc
           ) {
             const loc = getMainFunctionTokenLocation(duplicatingFunction, functions[i].parent, context);
-            context.report({
-              message: MESSAGE,
-              data: { line: String(originalFunction.loc.start.line) },
-              loc,
-            });
+            const originalFunctionLoc = getMainFunctionTokenLocation(originalFunction, functions[j].parent, context);
+            const secondaryLocations = [
+              issueLocation(originalFunctionLoc, originalFunctionLoc, "Original implementation"),
+            ];
+            report(
+              context,
+              {
+                message: MESSAGE,
+                data: { line: String(originalFunction.loc.start.line) },
```

Comment:
How about avoid using `data` for issues with secondaries? It works, but it's almost happy coincidence:)